src/hotspot/os_cpu/linux_aarch64/atomic_linux_aarch64.hpp

```diff
@@ -151,6 +151,11 @@ inline T Atomic::PlatformCmpxchg<4>::operator()(T volatile* dest,
   switch (order) {
   case memory_order_relaxed:
     stub = aarch64_atomic_cmpxchg_4_relaxed_impl; break;
+  case memory_order_release:
+    stub = aarch64_atomic_cmpxchg_4_release_impl; break;
+  case memory_order_seq_cst:
+  case memory_order_acq_rel:
```

Comment:
Swap to keep the strength hierarchy.

```suggestion
  case memory_order_acq_rel:
  case memory_order_seq_cst:
```